src/components/probe/index.ts

```diff
@@ -52,6 +52,13 @@ interface ProbeSendNotification extends Omit<ProbeStatusProcessed, 'statuses'> {
   probeState?: ServerAlertState
 }
 
+// global variable
+let responseCount = 0
```

Comment:
Having global variable in here can lead to race condition/mutex.
It would be hard for other developer(s) to understand what will happen if we are running multiple probe at the same time. Will it cause mutex/race condition problem or not.
Can we have comment (description) in here: why we choose to use global variable and what will happen to this global variable if we run multiple probe?

Reply:
we can say it is safe to use global variable here since the probe is processed sequentially as mentioned in probe/index.ts line 143: intentionally wait for a request to finish before processing next request in loop

i have tested it, and i have added it to the comment too